docker_test.go

```diff
@@ -331,9 +321,10 @@ func run(m *testing.M) (int, error) {
 	cleanup := &testhelper.Cleanup{}
 	defer cleanup.Run()
 
-	KafkaContainer, err = destination.SetupKafka(pool, cleanup, destination.WithLogger(&testLogger{
-		logger.NewLogger().Child("kafka"),
-	}))
+	KafkaContainer, err = destination.SetupKafka(pool, cleanup,
```

Comment:
Some of us are using Apple with a M1 chip (arm64) and both `bitnami/kafka` & `bitnami/zookeeper` don't provide a docker image for `arm64`. As a result these tests delay dramatically and ultimately fail when running on a M1 chip which is bad development experience. If we cannot find a solution to that, then we should disable these tests when `runtime.GOARCH == arm64`
@pChondros @mcchran

Reply:
Given that this work is already done and it would take too long to retest all configuration with different images, I think we should just wait till bitnami supports it: https://github.com/bitnami/charts/issues/7305

In the meantime what you are suggesting should do the trick (i.e. `if runtime.GOARCH == arm64 { t.Skip() }`.

About the tests being disabled on CI I think it is a mistake to do so. My guess is that all those tests running with `t.Parallel()` take a serious toll on the CI virtual that has limited capacity. I will test it without the parallelization (the tests will take longer to run but a lot less likely to timeout).